bin/daru-view

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'rake'
+
+path = `gem which daru/view`.split('/')[0...-1].join('/')
+
+parser = OptionParser.new do |opts|
+  opts.banner = "Usage: daru-view [options]"
+  # Update google charts javascript dependent files, from latest Builds on google developers website
+  opts.on('-g', '--update-googlecharts', 'Update googlecharts.js libraries') do
+    sh "curl -# http://www.google.com/jsapi -L --compressed -o #{path}/view/adapters/js/googlecharts_js/google_visualr.js"
+    sh "curl -# http://www.gstatic.com/charts/loader.js -L --compressed -o #{path}/view/adapters/js/googlecharts_js/loader.js"
+  end
+
+  # Update highcharts.js from latest Builds on Highcharts codebase: http://code.highcharts.com/
+  opts.on('-H', '--update-highcharts', 'Update highcharts.js libraries') do
+    sh "mkdir -p #{path}/view/adapters/js/highcharts_js/modules/"
```

Comment:
We must have modularity , to update just modules or just adapters or just particular file.

Reply:
I have a doubt, why will someone using daru-view care about updating adapters and modules separately or a specific file? I think it is a different issue of having updates in a granular manner.

Reply:
For example, if someone wants to use new features added in highcharts3d then user may just want to update the file `highcharts-3d.js` . Let me know if it is little tricky task. We will add it as TODO task .